README.md

```diff
@@ -31,12 +31,12 @@ This is all that is needed - Nextflow will automatically fetch the plugin code a
 > [!NOTE]
 > The snippet above will always try to install the latest version, good to make sure
 > that the latest bug fixes are included! However, this can cause difficulties if running
-> offline. You can pin a specific release using the syntax `nf-validation@2.0.0`
+> offline. You can pin a specific release using the syntax `nf-schema@2.0.0`
```

Comment:
Are we going straight in at `2.0.0`?

Reply:
Yes since it's just a rebrand from `nf-validation` it would make more sense to continue the versioning from there